tests/Doctrine/Tests/DBAL/Functional/Schema/SchemaManagerFunctionalTestCase.php

```diff
@@ -199,8 +199,10 @@ public function testListTableColumns()
         $this->_sm->dropAndCreateTable($table);
 
         $columns = $this->_sm->listTableColumns('list_table_columns');
+        $columns_keys = array_keys($columns);
```

Comment:
Only nitpick I have: Please use camelCase throughout our codebase as we stick to PSR-2 coding standards.